common/io/Serial.cpp

```diff
@@ -140,7 +143,41 @@ bool UIntToSpeedT(uint32_t value, speed_t *output) {
   return false;
 }
 
+#ifdef HAVE_FLOCK
+bool OpenAndFlock(const std::string &path, int oflag, int *fd) {
```

Comment:
I think we've generally gone for having this the other way round and returning false, rather than this way where the build would break when it only impacts a small fraction of the functionality.

Reply:
Ok, done.  Now it will try to open the device, but return false instead of locking it.